app/controllers/application_controller.rb

```diff
@@ -61,10 +61,13 @@ def user_state
 
   def log_safe_current_user(reload: false)
     if @log_safe_current_user.nil? || reload
-      @log_safe_current_user = current_user.dup
-      email = @log_safe_current_user["info"]&.fetch("email", "")
-      @log_safe_current_user.delete("info")
-      @log_safe_current_user["email_md5"] = Digest::MD5.hexdigest(email)
+      email = current_user["info"]&.fetch("email", "")
+
+      @log_safe_current_user = {
+        email_md5: Digest::MD5.hexdigest(email),
```

Comment:
is there any point having this given we have the 2 lines below? I don't see a scenario where we'd have to md5 every email address in the db to find the associated record

Reply:
I don't mind dropping it, through it was added due to 
https://trello.com/c/OhXYtbKP/1396-3x-users-incorrectly-unable-to-get-past-unknown-org-page

@misaka do we still need the it?

Reply:
I think it's fine to leave then if it is needed for the edge case